example/src/App.tsx

```diff
@@ -78,21 +59,18 @@ export const App: React.FC = () => {
           {"If you want to go to the previous step, press "}<BoldText>{"Previous.\n"}</BoldText>
         </Text>
       </TourBox>
-    )
+    ),
   }, {
     alignTo: Align.SCREEN,
     before() {
-      return new Promise<void>((resolve, reject) => {
+      return new Promise<void>(resolve => {
         Animated.spring(gap, {
           bounciness: 100,
           speed: 1,
           toValue: Dimensions.get("screen").height * 0.25,
-          useNativeDriver: false // Disabled as RNST does not use native driver either (for now)
+          useNativeDriver: false, // Translate animation not supported native by native driver
         })
-        .start(({ finished }) => finished
-          ? resolve()
-          : reject()
-        );
+        .start(() => resolve());
```

Comment:
Is this change ok, not rejecting in the `finished = false` case?

Reply:
Yes, it's ok. The `finish = false` means that the animation could have been cancelled or restarted, it doesn't mean the animation went wrong. Animations are frequently cancelled by user interaction so we don't want to unnecessarily reject the promise 🙂